### ISSUE TITLE
React-native link and no CocoaPods required?

### ISSUE THREAD

**Issue by afilp** [charlires/react-native-segment-analytics#16]
@charlires

react-native-segment-analytics is the only package that requires CocoaPods and this causes several issues on our workflow. We do not want to use CocoaPods.

Now we see that a "react-native link" commit has been made:
6c9bdd3

Is CocoaPods still necessary for some reason? If not can the instructions show what needs to be done with plain "react-native link"?

Thanks!

**Comment by sondremare** [comment]
Yes it is still needed. Segment themselves recommend/show how to integrate only via CocoaPods
